app/views/content/life-as-a-teacher/teaching-as-a-career/why-teach/_article.html.erb

```diff
@@ -20,9 +20,12 @@
 
     <h2 class="heading--box-blue">Use skills from your degree or previous career</h2>
 
-    <p>Teaching provides you with a great environment to use the skills and knowledge you’ve developed during a degree or a previous career. You’ll share your passion with your pupils and continue to learn as you teach. Find out more about <a href="/life-as-a-teacher/change-careers">changing careers to become a teacher</a>.</p>
+    <p>Teaching provides you with a great environment to use the skills and knowledge you’ve developed during a degree or a previous career. You’ll share your passion with your pupils and continue to learn as you teach.</p>
 
     <p><%= render 'content/shared/quotes/quote_why_teach_jack' %></p>
+    <p>You may already have valuable experience working with pupils in the classroom, such as being a teaching assistant. Your expertise working with pupils will make you a desirable candidate for teacher training, enabling you to take what you have learned so far and make a smooth transition into teaching.</p>
```

Comment:
As before, I'm not sure we can say this `will` make them a desirable candidate / will have a smooth transition?